pkg/deploy/clusterrolebinding.go

```diff
@@ -43,6 +45,16 @@ func SyncClusterRoleBindingToCluster(
 		return nil, err
 	}
 
+	diff := cmp.Diff(clusterRB, specCRB, pvcDiffOpts)
+	if len(diff) > 0 {
```

Comment:
Sorry, did you tested update flow? I meant migration from one version che to next version che with more permissions 
 verb set. I asked, because I see that for role.go we have ignore fields https://github.com/eclipse/che-operator/blob/2e0bfecd456e20e2c289533fe40f05e5c5d78f3e/pkg/deploy/role.go#L29 and I guess such fields need to be to in the clusterole too. But I'm not sure in 100%. What do you think @tolusha ?

Reply:
I did not, will try.

Reply:
`pvcDiffOpts` it is just a copy/paste
For `clusterrolebinding` we need a dedicates diffOpts.

Reply:
done